src/components/Router/CrossDeviceMobileRouter.tsx

```diff
@@ -204,16 +204,29 @@ export default class CrossDeviceMobileRouter extends Component<
         validEnterpriseFeatures?.hideOnfidoLogo
       ) {
         this.props.actions.hideOnfidoLogo(true)
-      } else {
+      } else if (
+        enterpriseFeatures.cobrand &&
+        validEnterpriseFeatures?.cobrand
+      ) {
         this.props.actions.hideOnfidoLogo(false)
 
         if (enterpriseFeatures.cobrand && validEnterpriseFeatures?.cobrand) {
```

Comment:
You don't need this if statement here anymore

Reply:
Thanks, updated.